{"metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"decapsulationKeyInconsistent"}],"roleHeading":"Case","role":"symbol","modules":[{"name":"SwiftKyber"}],"externalID":"s:10SwiftKyber0B9ExceptionO28decapsulationKeyInconsistentyA2CmF","title":"KyberException.decapsulationKeyInconsistent","symbolKind":"case"},"schemaVersion":{"patch":0,"minor":3,"major":0},"primaryContentSections":[{"declarations":[{"platforms":["macOS"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"decapsulationKeyInconsistent","kind":"identifier"}],"languages":["swift"]}],"kind":"declarations"}],"sections":[],"abstract":[{"type":"text","text":"Inconsistent decapsulation key data"}],"hierarchy":{"paths":[["doc:\/\/SwiftKyber\/documentation\/SwiftKyber","doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException"]]},"kind":"symbol","variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftkyber\/kyberexception\/decapsulationkeyinconsistent"]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException\/decapsulationKeyInconsistent"},"references":{"doc://SwiftKyber/documentation/SwiftKyber/KyberException/decapsulationKeyInconsistent":{"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"decapsulationKeyInconsistent"}],"url":"\/documentation\/swiftkyber\/kyberexception\/decapsulationkeyinconsistent","role":"symbol","type":"topic","abstract":[{"text":"Inconsistent decapsulation key data","type":"text"}],"kind":"symbol","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException\/decapsulationKeyInconsistent","title":"KyberException.decapsulationKeyInconsistent"},"doc://SwiftKyber/documentation/SwiftKyber":{"abstract":[{"type":"text","text":"Module-Lattice-Based Key-Encapsulation Mechanism Standard"}],"type":"topic","title":"SwiftKyber","role":"collection","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber","url":"\/documentation\/swiftkyber","kind":"symbol"},"doc://SwiftKyber/documentation/SwiftKyber/KyberException":{"role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"KyberException"}],"identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException","kind":"symbol","navigatorTitle":[{"text":"KyberException","kind":"identifier"}],"abstract":[{"type":"text","text":"The Kyber exceptions"}],"type":"topic","url":"\/documentation\/swiftkyber\/kyberexception","title":"KyberException"}}}